cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -437,6 +437,7 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 		// In dry run only utilization is updated
 		calculateUnneededOnly := scaleDownInCooldown || scaleDown.nodeDeletionTracker.IsNonEmptyNodeDeleteInProgress()
 
+		metrics.UpdateScalingDownStatus(!a.processorCallbacks.disableScaleDownForLoop)
```

Comment:
It seems what you want to capture is the `!scaleDownInCooldown`

Reply:
@losipiuk what do you think if I set this metric to `false` in these 2 codeblocks: [here](https://github.com/kubernetes/autoscaler/blob/8d9010e11eb01f0d8cb2f58ec53239240802eef7/cluster-autoscaler/core/static_autoscaler.go#L446) and [here](https://github.com/kubernetes/autoscaler/blob/8d9010e11eb01f0d8cb2f58ec53239240802eef7/cluster-autoscaler/core/static_autoscaler.go#L448)

And I set it to `true` [here](https://github.com/kubernetes/autoscaler/blob/8d9010e11eb01f0d8cb2f58ec53239240802eef7/cluster-autoscaler/core/static_autoscaler.go#L450)

Reply:
@losipiuk I've just updated the PR to propose this change 👍